apps/devhub/tests/test_views.py

```diff
@@ -1974,8 +1974,8 @@ def test_login_required(self):
     def test_create_fileupload(self):
         self.post()
 
-        upload = FileUpload.objects.get(name='animated.png')
-        eq_(upload.name, 'animated.png')
+        upload = FileUpload.objects.filter().order_by('-created')[0]
+        assert 'animated.png' in upload.name
```

Comment:
You can test the filename for real: `assert '{0}_animated.png'.format(upload.pk)`